src/Raven.Client/Documents/Operations/ETL/SQL/SqlConnectionString.cs

```diff
@@ -54,7 +54,11 @@ public override DynamicJsonValue ToJson()
 
         public override DynamicJsonValue ToAuditJson()
         {
-            return ToJson();
+            var json = base.ToJson();
+            
+            json[nameof(FactoryName)] = FactoryName;
```

Comment:
We could call `SqlConnectionStringParser.GetDatabaseAndServerFromConnectionString(FactoryName, ConnectionString);` and add database and server. This way it will be useful info but without any secrets

Reply:
i like this, added in the last commit (https://github.com/ravendb/ravendb/pull/19152/commits/3064e03c74465c03680814f6944d962c6cb34734)